### PR TITLE
Lvnod 46 User autorization

### PR DESCRIPTION
**Pull request by AlbaMontis**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-46)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-48)

## Code reviewers

- [x] @yurii-kokhalevych
- [x] @denisbodnar
- [x] @shyyka

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

### CODE REVIEW

**Review by denisbodnar — APPROVED** [review]

**Review by yurii-kokhalevych — APPROVED** [review]

**Review by shyyka — APPROVED** [review]

File: app/src/helper/constants.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const URL = {
+  LOGIN: 'http://localhost:8082/api/auth/login'
```
Comment (4asof4ik) [inline app/src/helper/constants.js:2]: Host and port

File: app/src/components/Login/Login.js [hunk 2]
```diff
@@ -19,40 +24,68 @@ export default class Login extends React.Component {
...
+  
+  user = () => ({
+    'email': this.state.email,
+    'password': this.state.password
+  })
 
   handleSubmit = (e) => {
+    const self = this;
     e.preventDefault();
+    request
```
Comment (4asof4ik) [inline app/src/components/Login/Login.js:44]: This should  be in action folder auth.action.js

File: app/src/components/Navigation/Navigation.js [hunk 3]
```diff
@@ -18,6 +37,9 @@ export default class Navigation extends React.Component {
               <NavItem eventKey={3}>Contacts</NavItem>
             </LinkContainer>
           </Nav>
+          {this.state.loggedIn &&
```
Comment (4asof4ik) [inline app/src/components/Navigation/Navigation.js:40]: Logout should be in dropdown with edit profile

File: app/src/components/Secret/Secret.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default class Secret extends React.Component {
```
Comment (4asof4ik) [inline app/src/components/Secret/Secret.js:3]: Why you create this comp...

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

File: app/src/components/Navigation/Navigation.js [hunk 5]
```diff
@@ -1,8 +1,27 @@
...
+  }
+  
+  componentWillMount () {
+    sessionStorage.getItem('token')?
+      this.setState({loggedIn: true}): this.setState({loggedIn: false})
+  }
+
+  logout = () => {
+    sessionStorage.removeItem('token');
+    window.location.reload();
```
Comment (4asof4ik) [inline app/src/components/Navigation/Navigation.js:22]: redirect to root path

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Pull request closed without merging**